components/scanners/trivy/vendor/github.com/smithy-security/pkg/sarif/sarif.go

```diff
@@ -205,8 +206,11 @@ func (s *SarifTransformer) transformToOCSF(toolName string, res *sarif.Result) (
 	rule := s.ruleToTools[*ruleID]
 	var cve *ocsf.Cve
 	if rule.FullDescription != nil {
-		uid := s.extractCVE(rule.FullDescription.Text)
-		if uid != "" {
+		if strings.HasPrefix(*ruleID, "CVE-") {
```

Comment:
this is a hotfix that is not present in the sarif to ocsf library and needs to be propagated soon to ensure correct CVE ID parsing